packages/jest-circus/src/__tests__/hooks_error.test.js

```diff
@@ -15,10 +15,12 @@ describe('hooks error throwing', () => {
   test.each([['beforeEach'], ['beforeAll'], ['afterEach'], ['afterAll']])(
```

Comment:
can change this to `describe.each`, then have the inner `for-loop` be `test.each`